dbtmetabase/__init__.py

```diff
@@ -86,7 +86,8 @@ class OptionAcceptableFromConfig(click.Option):
     more resilence to raising an error when the option exists in the users config.
 
     This also overrides default values for boolean CLI flags (e.g. --use_metabase_http/--use_metabase_https) in options when
-    no CLI flag is passed, but a value is provided in the config file (e.g. metabase_use_http: True)."""
+    no CLI flag is passed, but a value is provided in the config file (e.g. metabase_use_http: True).
```

Comment:
Revert all changes unrelated to the functionality you're implementing (e.g. removing newlines, moving comment quotes)

Reply:
Sorry, I discovered this was misalignment in formatter versions, which I have now fixed in the master.